### ISSUE TITLE
Mixed Light/Dark Theme similar to original bitterologist theme

### ISSUE THREAD

**Issue by JeanOlivier** [Ferdi265/numix-solarized-gtk-theme#19]
tl;dr: See what I mean on [this picture](https://pre00.deviantart.net/cccb/th/pre/f/2014/183/8/3/numix_solarized_by_bitterologist-d6wm3nc.png) on the nautilus window with the Dark top.

The original numix solarized theme by bitterologist includes a theme simply called "Solarized" that is a mixed of the "Dark/Light Blue" themes that you've made here. I personally prefer the mix of dark headers and light background that it has. Would it be possible to add such an option to this project?

I've attempted to do it myself, but I can't compile the project as it spits an error: `Failed to locate 'assets/checkbox-checked-dark.png' in any source directory.`

The bitterologist themes are outdated and have issues with small text zones and missing checkboxes, so it's not very useable.

Thank you very much for considering it or helping me compiling my own version!

PS: I also noticed that the red color from numix is still used for the windows buttons (close/minimize/maximize) under Ubuntu 17.10 using your latest release, strangely enough it does that only on the "big" windows buttons, not on the small ones that are on some apps like nautilus/settings.

**Comment by Ferdi265** [comment]
1. Bitterlologist's version was mostly custom-made and doesn't use the upstream Numix code (or a really old version). This one is forked off Numix. I don't quite know how to replicate exactly that color scheme. (I don't really know much about GTK themes)

2. The compile error looks weird, I'll look into it

3. Windows like nautilus have "Client side decoration" and use the GTK theme for decoration colors. The other windows are decorated by the window manager (Unity), and the WM-specific parts of the theme aren't updated to solarized colors yet (see the unity folder in src)

**Comment by treymerkley** [comment]
Unless Ferdi just wants to build this theme, you may want to check out [SolArc](https://github.com/schemar/solarc-theme). It seems like it has a color scheme closer to what you're describing than Numix Solarized.

**Comment by JeanOlivier** [comment]
Thanks for the suggestion. I tested SolArc and it's close to what I want, yet not quite there. At least I can compile it from sources properly, so I should be able to mod it to my needs. There are still some colors not changed by SolArc, like the chrome grey-header, which makes it look weird, and the window close button has weird contrast issues.

All in all I'd still prefer Numix-Solarized if I could adapt it to my preferences.  @Ferdi265, any idea on how to fix my compilation issue?

**Comment by Ferdi265** [comment]
@JeanOlivier, I can't seem to reproduce your issue. A fresh install from a fresh clone of the repo seems to work for me.

I'll try it again on a fresh arch once I find time to do that (in case it's a missing dependency or something), but I really don't know why that is.

Can I have a full error log for debugging purposes? (just make a gist or pastebin)

**Comment by JeanOlivier** [comment]
I managed to compile it in the end. I don't know why it wasn't working properly, maybe a reboot fixed it? I was installing my new system so maybe I installed a missing dependency too.
EDIT: I still get the error from time to time. I have to execute `make clean` to fix it.

I'll try and tweak it to my taste, I might send you a pull request if I get a satisfactory result.

Thanks for answering my questions!

~~EDIT: In fact, in LightBlue, changing `@DARK_BG_COLOR_LIGHT@` to `#002b36` seem to fix most of my issues with the theme. All I need to figure out now is how to change the inactive scrollbar color (e.g. side of terminal in Ubuntu).~~

EDIT: I'm done with my `SolarizedLightBlueDarkTop` modification, everything is perfect to my taste except the unity decorations. Do you want me to submit a PR?

**Comment by Ferdi265** [comment]
A PR sounds good :)
